wasm-build.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
 
-wasm-pack build -t nodejs -d pkg-node --out-name flux-lsp-node --scope influxdata --release
-wasm-pack build -t browser -d pkg-browser --out-name flux-lsp-browser --scope influxdata --release
+EXTRA_ARGS="--"
+if [[ ! -z "${LSP2}" ]]; then
+    echo "Building with --featurs lsp2"
+    EXTRA_ARGS="${EXTRA_ARGS} --features lsp2"
+fi
+
+wasm-pack build -t nodejs -d pkg-node --out-name flux-lsp-node --scope influxdata --release $EXTRA_ARGS
+wasm-pack build -t browser -d pkg-browser --out-name flux-lsp-browser --scope influxdata --release $EXTRA_ARGS
```

Comment:
What happens when we run this script with the feature flag enabled? Based on our earlier conversation, I would expect this build for the browser target to fail. Or does the incompatibility between `tokio` and wasm only manifest at runtime?

Reply:
If you do `LSP2=1 ./wasm-build.sh`, it won't fail, but it won't generate a wasm that will pass the integration tests currently. There's a comment in the circleci config to that effect.